app/commands/feeds/find_new_stories.rb

```diff
@@ -16,9 +16,7 @@ def new_stories
     @raw_feed.entries.each do |story|
       break if @latest_entry_id && story.id == @latest_entry_id
 
-      unless story_age_exceeds_threshold?(story) || StoryRepository.exists?(story.id, @feed_id)
-        stories << story
-      end
+      stories << story unless story_age_exceeds_threshold?(story) || StoryRepository.exists?(story.id, @feed_id)
```

Comment:
I'm thinking extracting the conditional to a guard clause might improve readabilty. Thoughts?
```rb
@raw_feed.entries.each do |story|
  break if @latest_entry_id && story.id == @latest_entry_id
  next if story_age_exceeds_threshold?(story) || StoryRepository.exists?(story.id, @feed_id)
  stories << story
end
```